config/settings/local.py

```diff
@@ -123,6 +123,15 @@
 CONNECTED_APP_CLIENT_SECRET = env("CONNECTED_APP_CLIENT_SECRET")
 CONNECTED_APP_CALLBACK_URL = env("CONNECTED_APP_CALLBACK_URL")
 
+
+AUTOSCALERS = {
+    "local-app": {
+        "app_name": "local-app",
+        "worker_type": "worker",
```

Comment:
app_name and worker_type aren't actually needed for the base Autoscaler class, right?

Reply:
Yes, this is correct.